src/proof/proof_manager.cpp

```diff
@@ -30,6 +30,7 @@
 #include "proof/theory_proof.h"
 #include "smt/smt_engine.h"
 #include "smt/smt_engine_scope.h"
+#include "smt/smt_statistics_registry.h"
```

Comment:
This shouldn't be required here since it's already included in `proof/proof_manager.h`

Reply:
Hmm, I don't see there. Am I missing something? We only seem to include `util/statistics_registry.h` there.

Reply:
You are right! Must have misread...